src/Nancy.Swagger.Annotations/SwaggerObjects/AnnotatedResponse.cs

```diff
@@ -9,7 +9,8 @@ namespace Nancy.Swagger.Annotations.SwaggerObjects
 {
     public class AnnotatedResponse : global::Swagger.ObjectModel.Response
     {
-        public int StatusCode { get; set; }
+        private int StatusCode { get; set; }
+        public int GetStatusCode() => StatusCode;
```

Comment:
See comment above, but the public property `StatusCode` gets dropped into the swagger JSON, e.g. `"StatusCode": xxx`